sh_scrapy/stats.py

```diff
@@ -1,30 +1,20 @@
-import time
 from twisted.internet import task
 from scrapy.statscol import StatsCollector
+
 from sh_scrapy import hsref
+from sh_scrapy.writer import pipe_writer
 
 
 class HubStorageStatsCollector(StatsCollector):
 
-    # TODO: make this configurable per project
-    STATS_TO_COLLECT = [
-        'item_scraped_count',
-        'response_received_count',
-        'scheduler/enqueued',
-        'scheduler/dequeued',
-        'log_count/ERROR',
-    ]
     INTERVAL = 30
 
     def __init__(self, crawler):
         super(HubStorageStatsCollector, self).__init__(crawler)
         self.hsref = hsref.hsref
 
     def _upload_stats(self):
-        row = [int(time.time() * 1000)]
-        row.extend(self._stats.get(x, 0) for x in self.STATS_TO_COLLECT)
-        self.hsref.job.samples.write(row)
```

Comment:
Sorry, I don't recall all the details about samples probably, are we going to support it in the new version of ep-scrapy? I see it's supported in kumo-forwarder